app/controllers/api/databases.php

```diff
@@ -2524,31 +2524,15 @@ function updateAttribute(
     ->inject('dbForProject')
     ->action(function (string $databaseId, string $collectionId, string $key, Response $response, Database $dbForProject) {
 
-        $database = Authorization::skip(fn() => $dbForProject->getDocument('databases', $databaseId));
-
-        if ($database->isEmpty()) {
-            throw new Exception(Exception::DATABASE_NOT_FOUND);
-        }
-        $collection = $dbForProject->getDocument('database_' . $database->getInternalId(), $collectionId);
-
-        if ($collection->isEmpty()) {
-            throw new Exception(Exception::COLLECTION_NOT_FOUND);
-        }
```

Comment:
But these are helpful to give feedback that the databaseId and collectionId passed in are correct 🧐 

Also, the `getDocument()` calls here aren't too bad because `getDocument()` goes to the cache rather than the database.

Reply:
Got it, added them back